api/server/server.go

```diff
@@ -214,11 +214,21 @@ func postContainersPause(eng *engine.Engine, version version.Version, w http.Res
 	if err := parseForm(r); err != nil {
 		return err
 	}
-	job := eng.Job("pause", vars["name"])
-	if err := job.Run(); err != nil {
+
+	name := vars["name"]
```

Comment:
@crosbymichael I've created an issue https://github.com/docker/docker/issues/12251 but I can't label it beginner or something